<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-34TR8MB" failures="1" tests="5" name="Test" time="114.335" errors="0" timestamp="2022-01-05T12:52:18 IST">
  <testcase classname="Testautomation.Testng" name="loginwithvalidcred" time="67.656"/>
  <testcase classname="Testautomation.Testng" name="logninvalid2" time="1.017">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method Testautomation.Testng.logninvalid2() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method Testautomation.Testng.logninvalid2() didn't finish within the time-out 1000
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- logninvalid2 -->
  <testcase classname="Testautomation.Testng" name="login" time="0.0">
    <skipped/>
  </testcase> <!-- login -->
  <testcase classname="Testautomation.Testng" name="logout" time="0.004"/>
  <testcase classname="Testautomation.Testng" name="logininvalid" time="45.429"/>
  <testcase classname="Testautomation.Testng" name="logninvalid1" time="0.0">
    <ignored/>
  </testcase> <!-- logninvalid1 -->
</testsuite> <!-- Test -->
